tested/testsuite.py

```diff
@@ -581,7 +581,7 @@ class Testcase(WithFeatures, WithFunctions):
     input: Statement | MainInput | LanguageLiterals
     description: Message | None = None
     output: Output = field(factory=Output)
-    link_files: list[FileUrl] = field(factory=list)
+    link_files: list[InputFile] = field(factory=list)
```

Comment:
Would seem logic to also rename `link_files` to `input_files` as it's a list of input file.